### PR TITLE
Add Production Build 

### LINKED ISSUE #58: Add production app build in main Dockerfile
The `Dockerfile.dev` can use the development server, but the production image should have a prebuild app:

- need production build, see https://reactjs.org/docs/optimizing-performance.html
- maybe https://react-server.io/docs/guides/production ?

Related to #57

### PR DESCRIPTION
**Pull request by Fmazin**
This pull request should resolve #58.

### CODE REVIEW

**Comment by Fmazin** [comment]
As i discussed in #58 the multistage dockerfile for the production build for the ui container seems to work. But when the when the whole docker-compose is started the startpage will throw a bad `502 Bad Gateway`.

File: README.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ The platform is available at [http://localhost](http://localhost) and the API at
 
 ## Production environment with docker-compose
 
-This project has another docker-compose configuration for the deployment of a production build (file `ui/docker-compose-prod.yml`). This configuration has no `ui` container. Instead the webserver container creates a static production build with the command [`npm run build`](https://create-react-app.dev/docs/available-scripts/).  Which then hosted through nginx. For this reason there is also a another nginx configuration (file `ui/dev/nginx-prod.conf`) with the corresponding 
+This project has another docker-compose configuration for the deployment of a production build (file `ui/docker-compose-prod.yml`). This configuration has no `ui` container. Instead the webserver container creates a static production build with the command [`npm run build`](https://create-react-app.dev/docs/available-scripts/) using a [multi-stage docker file](/usr/share/hunspell) (file `ui/Dockerfile_pb_2`).  Which then hosted through nginx. For this reason there is also a another nginx configuration (file `ui/dev/nginx-prod.conf`) with the corresponding
```
Comment (nuest) [inline README.md:35]: The for the multi-stage Docker file seems to be broken. Please also capitalise "Docker" (it's a name).

"Which then hosted through nginx." is not a full sentence.

File: ui/dev/mime.types [hunk 2]
```diff
@@ -0,0 +1,13 @@
+types {
+    text/html                             html htm shtml;
+    text/css                              css;
+    text/xml                              xml;
+    image/gif                             gif;
+    image/jpeg                            jpeg jpg;
+    application/x-javascript              js;
+    application/atom+xml                  atom;
+    application/rss+xml                   rss;
+    ..........................................
```
Comment (nuest) [inline ui/dev/mime.types:10]: Suggest to remove the placeholder.

Are these _all_ mime types used by the UI, or maybe there is a more extensive list just to be sure?

**Review by nuest — CHANGES_REQUESTED** [review]

**Review by nuest — APPROVED** [review]
I could run this locally after forcing a rebuild of the webserver image.

Good work! :100:

**Comment by nuest** [comment]
@NJaku01 Please try out the new compose configurations - the prerendered UI seems to be noticeably quicker.

Note that you have to use a different `docker-compose` command now to run the development version of the UI - see `README.md` !

**Pull request merged**